client/tsconfig.json

```diff
@@ -19,7 +19,6 @@
       "dom"
     ],
     "module": "esnext",
-    "baseUrl": "./",
-    "resolveJsonModule": true
```

Comment:
is this not needed to import the json files?

Reply:
it's a repeat of line four, the linter spotted it.